pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -1,26 +1,31 @@
 {
-  lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonOlder,
-  pytestCheckHook,
+  fetchpatch,
+  lib,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   gdcm,
   nibabel,
   numpy,
+  pillow,
```

Comment:
I don't see where `pillow` is needed as a dependency?

Reply:
In its absence the tests fail. I should have included it in the nativeCheckInputs. fixing it